src/main/java/org/sonarlint/intellij/core/SonarLintEngineFactory.java

```diff
@@ -61,34 +65,25 @@
   private static final Set<Language> CONNECTED_ADDITIONAL_LANGUAGES = EnumSet.of(
     Language.SCALA,
     Language.SWIFT,
-    Language.XML
-  );
+    Language.XML);
 
   ConnectedSonarLintEngine createEngine(String connectionId) {
     Set<Language> enabledLanguages = EnumSet.copyOf(STANDALONE_LANGUAGES);
     enabledLanguages.addAll(CONNECTED_ADDITIONAL_LANGUAGES);
 
     amendEnabledLanguages(enabledLanguages);
 
-    GlobalLogOutput globalLogOutput = SonarLintUtils.getService(GlobalLogOutput.class);
-    ModulesRegistry modulesRegistry = SonarLintUtils.getService(ModulesRegistry.class);
-    final NodeJsManager nodeJsManager = SonarLintUtils.getService(NodeJsManager.class);
     URL cFamilyPluginUrl = findEmbeddedCFamilyPlugin(getPluginsDir());
-    ConnectedGlobalConfiguration.Builder config = ConnectedGlobalConfiguration.builder()
-      .setLogOutput(globalLogOutput)
-      .setSonarLintUserHome(getSonarLintHome())
+    ConnectedGlobalConfiguration.Builder configBuilder = ConnectedGlobalConfiguration.builder()
       .addEnabledLanguages(enabledLanguages.toArray(new Language[0]))
-      .setExtraProperties(prepareExtraProps())
-      .setNodeJs(nodeJsManager.getNodeJsPath(), nodeJsManager.getNodeJsVersion())
-      .setWorkDir(getWorkDir())
-      .setModulesProvider(() -> modulesRegistry.getModulesForEngine(connectionId))
```

Comment:
This got lost in the refactoring to `configureCommonEngine`, the management of modules is not the same for connected engines vs the standalone engine.

Reply:
Oups, good catch indeed!